bloom/generators/release.py

```diff
@@ -53,8 +53,8 @@
 
 try:
     import catkin_pkg
-    from pkg_resources import parse_version
-    if parse_version(catkin_pkg.__version__) < parse_version('0.3.8'):
+    from packaging import version
+    if version.parse(catkin_pkg.__version__) < version.parse('0.3.8'):
```

Comment:
@cottsay @wjwwood what do you think about removing these inline version checks entirely? bloom now requires catkin-pkg >= 0.4.3 according to `setup.py` and `stdeb.cfg` so this is just one more place where version dependencies have to be managed.

Reply:
I think that's probably fine, but I think historically I think this was in place to catch things like catkin_pkg is from pip and bloom is from apt or something like that, where the requirements can be circumvented. And this just a slightly nicer error than you'd get otherwise. Maybe we could extract this automatically some how so that you wouldn't need to update it manually.